views/editor/bramble.html

```diff
@@ -113,16 +123,22 @@
                 transform: rotateX(-360deg) rotateY(-360deg);
             }
         }
+
         </style>
 
     <div id="spinner-container">
         <div class="thimble-spinner"></div>
-        <div class="loading-message message">{{ gettext("loadingProject") }}</div>
+          <div class="loading-message message">
+           {{ gettext("loadingProject") }}
+	  <div class="loading-message-help-link">
+            <p>{{ gettext("loadingProjectSupportSuggestion") | safe }}</p>
+    	  </div>
+	 </div>
         <div class="error-message message">
             <p>{{ gettext("errorLoadingProject") | safe }}</p>
             <p>{{ gettext("errorLoadingProjectSuggestion") }}</p>
-        </div>
-    </div>
+       </div>
+     </div>
```

Comment:
Hi @evantsyferov - please fix up the indentation in the block from line 129 to 141. Please use 2 space indentation, and not tab characters.

Here's how the code appears right now in my editor...

![image](https://cloud.githubusercontent.com/assets/25212/23631696/12685a30-0274-11e7-8b1f-b7d8568ebddd.png)

Notice that the indentation is a mix of spaces and tabs. Please match the indentation so that it looks similar to the ``id=browser-support-warning`` code block below. 

Thanks!